sdk/azidentity/azure_cli_credential.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// Enables authentication to Azure Active Directory using Azure cli to generated an access token.
+type AzureCliCredential struct {
```

Comment:
This should be AzureCLICredential. CLI should be in all caps wherever it appear to follow convention.

Reply:
Let’s hold on changing this until we get consensus across Langs as it is Cli across all right now.  Thanks.

Reply:
This is a Golang specific requirement. It's idiomatic to the language to use common casing when using acronyms. In this, we differ from the rest of the languages, since the Gopher community and the linter look for this. The current go linter already complains for common acronyms such as ID, which is why we have tenantID and not tenantId like other languages.

Reply:
Gotcha.  Thanks for the explanation.  @tzhanl, @wantedfast this if for Go only, change to CLI.  Thank you.